.circleci/config.yml

```diff
@@ -94,12 +94,12 @@ jobs:
     <<: *shared
     docker:
       - image: glpi/circleci-env-core:php_7.0_fpm-node
-      - image: circleci/mariadb:10.2
+      - image: circleci/mariadb:10.1
```

Comment:
As far as I remember there are changes in 10.2 that causes issues, this one should stay tested

Reply:
I put 10.1 here as 5.6 test have been dropped in 9.5.
10.2 is now with PHP 7.1 so will still be launched on nightly tests.

Reply:
I reverted this change.